tools/distpartitioning/dataset_utils.py

```diff
@@ -48,60 +48,171 @@ def get_dataset(input_dir, graph_name, rank, world_size, schema_map):
         edge-type may have several edge features and associated tensor data.
```

Comment:
this function is to read the entire dataset? if so, can you provide more information in the docstring. for example, what data is read?

Reply:
Added very detailed description in the comments about nodes, edges and node-types and their assocaited data etc...